src/api/auth.ts

```diff
@@ -407,7 +407,8 @@ export function publishPackage(
                 owners: [ username ],
                 organization: organization ? organization : '',
                 teamPermissions: permissions,
-                memberPermissions: [{ name: username, permission: 'read-write' }]
+                memberPermissions: [{ name: username, permission: 'read-write' }],
```

Comment:
Actually, I don't like either :), but yeah I think your's version is a little nicer... I'll change into that format....